.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libsodium"]
 	path = libsodium
-	url = git://github.com/jedisct1/libsodium.git
+	url = https://github.com/jedisct1/libsodium.git
```

Comment:
This seems to be a mistake

Reply:
Ah I see your commit now. This has been working for us previously tho

Reply:
I initialy had some problems updating the submodule and switching to http solved the issue. I tried to reproduce this issue just now, but I failed. If you want I can remove the respective commit, but I think it would not hurt, at least when reading [the documentation](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols).

Reply:
I can just cherrypick the other commits when I merge

Reply:
I removed it.